### PR TITLE
[UT]: add ut for pkg/apis/events/v1/conversion.go

### PR DESCRIPTION
**Pull request by taoyounger**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:

add ut for pkg/apis/events/v1/conversion.go

Which issue(s) this PR fixes:
Fixes #

Special notes for your reviewer:
Does this PR introduce a user-facing change?

```
None
```

Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```
None
```

### CODE REVIEW

**Comment by jiahuif** [comment]
/assign @charles-chenzz 
Thank you.
/triage accepted

File: pkg/apis/events/v1/conversion_test.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+					},
+					DeprecatedSource: corev1.EventSource{
+						Component: "controller-manager",
+						Host:      "master1",
+					},
+					Note: "convert v1 event to core event",
+				},
+				out: &core.Event{},
+			},
+			wantErr: false,
```
Comment (aojea) [inline pkg/apis/events/v1/conversion_test.go:62]: this seems to test only happy path? I didn't check to the code but are not any case that fails we need to test?

**Pull request closed without merging**